### PR TITLE
Creating KMS Key Class

### PR DESCRIPTION
**Pull request by fuddstar**
Adding doco, unit tests, systems test for KMS to create a file

**BUILD PASSING** - https://gaautobots.atlassian.net/builds/browse/AM-AUT27-10

### CODE REVIEW

**Comment by GypsyBojangles** [comment]
Approved

**Comment by GypsyBojangles** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/GeoscienceAustralia/amazonia/pull-request/49/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=GeoscienceAustralia/amazonia)

File: amazonia/classes/kms.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python3
+
+from troposphere import Output, Join, Ref
+from troposphere.kms import Key
+
+
+class KmsKey(object):
+    def __init__(self, key_title, key_rotation, key_admins, key_users, template):
+        """
```
Comment (snayler0) [inline amazonia/classes/kms.py]: docstrings for the params?

Comment (fuddstar) [inline amazonia/classes/kms.py]: Added

**Comment by GypsyBojangles** [comment]
Approved

**Comment by snayler0** [comment]
Approved

**Pull request merged**